api/api.proto

```diff
@@ -706,6 +732,8 @@ message VolumeSpec {
   bool near_sync = 51;
   // NearSyncReplicationStrategy is replication strategy for near sync volumes
   NearSyncReplicationStrategy near_sync_replication_strategy = 52;
+  // shared_block is true if this volume can be concurrently accessed by multiple nodes
+  bool shared_block = 54;
```

Comment:
should we just add access mode enum member instead of another bool ? (shared, sharedv4, shared_block ?)

Reply:
+1 on enum. Today RWO raw block is dictated through FS field set to None instead of an explicit field for access.

Reply:
Lots of this code is carried over for legacy reasons. I will convert shared_block to an enum, but will not touch other pieces of code that want to use this field.